src/styles/blocks/card.css

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background-color: #fff;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.card__img {
+  align-self: center;
+
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin: 40px 0 0;
```

Comment:
I think it's better to reset default margins for headings and paragraphs first, and then apply only margin top here, it will be much readable

Also you already have `reset.css` file and you reset body margins there, why not add some tags to that rule 🙂

Reply:
Great idea)
Done!